File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/BillingAddCcRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Request to add a new creditcard into the system.
  /// </summary>
  [DataContract]
  public class BillingAddCcRequest {
    /// <summary>
    /// Gets or Sets Name
    /// </summary>
    [DataMember(Name="name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }

    /// <summary>
    /// Gets or Sets Address
    /// </summary>
    [DataMember(Name="address", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "address")]
    public string Address { get; set; }

    /// <summary>
    /// Gets or Sets City
    /// </summary>
    [DataMember(Name="city", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "city")]
    public string City { get; set; }

    /// <summary>
    /// Gets or Sets State
    /// </summary>
    [DataMember(Name="state", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "state")]
    public string State { get; set; }

    /// <summary>
    /// Gets or Sets Country
    /// </summary>
    [DataMember(Name="country", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "country")]
    public string Country { get; set; }

    /// <summary>
    /// Gets or Sets Zip
    /// </summary>
    [DataMember(Name="zip", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "zip")]
    public string Zip { get; set; }

    /// <summary>
    /// Gets or Sets Cc
    /// </summary>
    [DataMember(Name="cc", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "cc")]
    public string Cc { get; set; }

    /// <summary>
    /// Gets or Sets CcExp
    /// </summary>
    [DataMember(Name="cc_exp", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "cc_exp")]
    public string CcExp { get; set; }

    /// <summary>
    /// Gets or Sets CcCcv2
    /// </summary>
    [DataMember(Name="cc_ccv2", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "cc_ccv2")]
    public string CcCcv2 { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class BillingAddCcRequest {\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Address: ").Append(Address).Append("\n");
      sb.Append("  City: ").Append(City).Append("\n");
      sb.Append("  State: ").Append(State).Append("\n");
      sb.Append("  Country: ").Append(Country).Append("\n");
      sb.Append("  Zip: ").Append(Zip).Append("\n");
      sb.Append("  Cc: ").Append(Cc).Append("\n");
      sb.Append("  CcExp: ").Append(CcExp).Append("\n");
      sb.Append("  CcCcv2: ").Append(CcCcv2).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/billing_add_cc_request.dart
part of swagger.api;

class BillingAddCcRequest {
  
  String name = null;

  String address = null;

  String city = null;

  String state = null;

  String country = null;

  String zip = null;

  String cc = null;

  String ccExp = null;

  String ccCcv2 = null;

  BillingAddCcRequest();

  @override
  String toString() {
    return 'BillingAddCcRequest[name=$name, address=$address, city=$city, state=$state, country=$country, zip=$zip, cc=$cc, ccExp=$ccExp, ccCcv2=$ccCcv2, ]';
  }

  BillingAddCcRequest.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    name = json['name'];
    address = json['address'];
    city = json['city'];
    state = json['state'];
    country = json['country'];
    zip = json['zip'];
    cc = json['cc'];
    ccExp = json['cc_exp'];
    ccCcv2 = json['cc_ccv2'];
  }

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'address': address,
      'city': city,
      'state': state,
      'country': country,
      'zip': zip,
      'cc': cc,
      'cc_exp': ccExp,
      'cc_ccv2': ccCcv2
     };
  }

  static List<BillingAddCcRequest> listFromJson(List<dynamic> json) {
    return json == null ? new List<BillingAddCcRequest>() : json.map((value) => new BillingAddCcRequest.fromJson(value)).toList();
  }

  static Map<String, BillingAddCcRequest> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, BillingAddCcRequest>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new BillingAddCcRequest.fromJson(value));
    }
    return map;
  }
}
